src/main.c

```diff
@@ -148,13 +148,50 @@ struct Bilhete* sessao_salvar_bilhete(
   bool meia_entrada,
   bool combo
 ) {
-  // @todo: Implementar função para salvar bilhete em uma sessão.
-  return NULL;
+    unsigned int valor_p, preco_b, lugar_c;
+    char lugar_l, filme_n [65];
+    bool meia, combo_use;
+
+    sessao->bilhetes[sessao->indice_bilhete].id = sessao->indice_bilhete + 1;
+    sessao->bilhetes[sessao->indice_bilhete].sala->filme[65] = filme_n;
+    sessao->bilhetes[sessao->indice_bilhete].sala->preco_bilhete = preco_b;
+    sessao->bilhetes[sessao->indice_bilhete].valor_pago = valor_p;
+    sessao->bilhetes[sessao->indice_bilhete].lugar_coluna = lugar_c;
+    sessao->bilhetes[sessao->indice_bilhete].lugar_linha = lugar_l;
+    sessao->bilhetes[sessao->indice_bilhete].meia_entrada = meia;
+    sessao->bilhetes[sessao->indice_bilhete].combo = combo_use;
```

Comment:
A variável `combo_use` **não foi inicializada**. E por que você tá definindo o `combo` para ela? Você não tá recebendo esse valor como argumento desta função? Então não precisa de mais nenhuma variável, é só literalmente:

```suggestion
    sessao->bilhetes[sessao->indice_bilhete].combo = combo;
```